sadcowpng/sharedspaces/views.py

```diff
@@ -1,6 +1,12 @@
 from django.shortcuts import render
 from django.template import loader
 from django.http import HttpResponse
+from django.http import HttpResponseRedirect
```

Comment:
nit: This import comes from the same module as the import above, so you could put it on the same line. I think that's technically best practice, along with alphabetizing imports but again I have all of GeoDjango to learn tonight so I don't have room to nitpick too much.

Reply:
Condensed it down a bit more.